Facebooky/StringPair.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Facebooky
+{
+	public class StringPair
+	{
+		public string Key {get; set;}
+		public string Value { get; set; }
+
+		public StringPair(string i_Key, string i_Value)
+		{
+			Key = i_Key;
+			Value = i_Value;
+		}
+		public StringPair() { }
```

Comment:
set `Key` and `Value` to be `string.Empty` so that they are always valid strings.